extensions/amazon-lambda/runtime/src/main/java/io/quarkus/amazon/lambda/runtime/AmazonLambdaRecorder.java

```diff
@@ -45,14 +47,22 @@
     public void setHandlerClass(Class<? extends RequestHandler<?, ?>> handler, BeanContainer container) {
         handlerClass = handler;
         beanContainer = container;
-        objectMapper = new ObjectMapper()
+        AmazonLambdaRecorder.objectMapper = getObjectMapper()
                 .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
                 .configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_PROPERTIES, true);
         Method handlerMethod = discoverHandlerMethod(handlerClass);
         objectReader = objectMapper.readerFor(handlerMethod.getParameterTypes()[0]);
         objectWriter = objectMapper.writerFor(handlerMethod.getReturnType());
     }
 
+    private ObjectMapper getObjectMapper() {
+        InstanceHandle<ObjectMapper> instance = Arc.container().instance(ObjectMapper.class);
+        if (instance.isAvailable()) {
+            return instance.get().copy();
```

Comment:
Why `copy()`?

Reply:
Because the recorder then adds its own settings which we don't want to propagate to the configured bean since that would affect all consumers